packages/frontend/src/Build.svelte

```diff
@@ -1086,9 +1088,18 @@ $: if (availableArchitectures) {
           <Button class="w-full" disabled={true}>Creating build task</Button>
         {:else}
           <Button on:click={buildBootcImage} disabled={errorFormValidation !== undefined} class="w-full">Build</Button>
+
+          <!-- macOS and Linux you can use Macadam / VM creation, show a disclaimer as a tip that you should add your public SSH key to test the VM. -->
+          <!-- On macOS and Linux, show tip about SSH key for VM testing -->
+          {#if isMac || isLinux}
+            <p class="text-sm text-[var(--pd-content-text)] m-0 pt-0" data-testid="vm-disclaimer">
+              To test the virtual machine, add your public SSH key under <b>Users</b> in the interactive build config. Set the output format to RAW or QCOW2.
+            </p>
+          {/if}
+
           <!-- If on Linux, warn that during the build, credentials will be asked in order to run an escalated privileged build prompt -->
           {#if isLinux}
-            <p class="text-sm text-[var(--pd-content-text)] pt-1">
+            <p class="text-sm text-[var(--pd-content-text)] m-0 pt-0">
```

Comment:
FYI, I changed the margin as well as the padding so it looks nicer. The default (from text-sm) was adding too much spacing.